dataprofiler/tests/profilers/profiler_options/test_boolean_option.py

```diff
@@ -87,3 +93,15 @@ def test_eq(self):
         self.assertNotEqual(options, options2)
         options2.is_enabled = False
         self.assertEqual(options, options2)
+
+    def test_json_encode(self):
```

Comment:
fixed this to use `get_options` otherwise it wouldn't raise errors for new tests that weren't properly updated